runtimes/protocol/lsp.ts

```diff
@@ -97,6 +97,14 @@ export interface AWSInitializationOptions {
      * and is extected to maintain data across updates and re-installations.
      */
     clientDataFolder?: string
+    /**
+     * The client signals AWS capabilities it supports.
+     */
+    awsClientCapabilities?: {
+        window?: {
+            showNotification?: boolean
```

Comment:
nit: should this be called just `notifications`, given this capability supports more than showing them, but also follow-ups (and potentially more in future)

Reply:
This setting will impact only `showNotification` - followups are not possible if there was no notification sent. I got a perception that capabilities in LSP are called after the methods that they enable. But I can rename it for cases when some other methods are added as part of notification.

Reply:
LSP is not consistent with capabilities naming, unfortunately